doc/radosgw/multisite.rst

```diff
@@ -801,10 +801,10 @@ are explicitly provided.
 Make a Realm the Default
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-One realm in the list of realms should be the default realm. There may
-be only one default realm. If there is only one realm and it wasn’t
-specified as the default realm when it was created, make it the default
-realm. Alternatively, to change which realm is the default, execute:
+One realm in the list of realms should be the default realm. There may be only
```

Comment:
s/may/can/

Reply:
Yeah, you're right about this. I'll make this change in a later grammar PR (to prevent scope creep here).